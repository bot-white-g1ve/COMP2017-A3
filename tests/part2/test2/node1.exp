Connection established with peer
Disconnected from peer
