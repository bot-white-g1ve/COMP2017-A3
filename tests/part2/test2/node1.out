Unable to connect to request peer
Unknown peer, not connected
